src/olympia/addons/tests/test_views.py

```diff
@@ -1299,12 +1299,12 @@ def test_set_disabled(self):
         assert addon.is_disabled is True
         assert addon.disabled_by_user is True  # sets the user property
 
-    @override_settings(EXTERNAL_SITE_URL='https://amazing.site')
+    @override_settings(DOMAIN='amazing.site')
     def test_set_homepage_support_url_email(self):
         data = {
-            'homepage': {'ro': '#%^%&&%^&^&^*'},
+            'homepage': {'en-US': '#%^%&&%^&^&^*'},
             'support_email': {'en-US': '#%^%&&%^&^&^*'},
-            'support_url': {'fr': '#%^%&&%^&^&^*'},
+            'support_url': {'en-US': '#%^%&&%^&^&^*'},
```

Comment:
this mix of locales was on purpose to make sure l10n was working, so doesn't changing it to 'en-US` reduce our coverage?